plugins/woocommerce/client/blocks/assets/js/blocks/store-notices/block.json

```diff
@@ -4,15 +4,13 @@
 	"title": "Store Notices",
 	"description": "Display shopper-facing notifications generated by WooCommerce or extensions.",
 	"category": "woocommerce",
-	"keywords": [
-		"WooCommerce"
-	],
+	"keywords": [ "WooCommerce" ],
 	"supports": {
+		"interactivity": {
+			"clientNavigation": false
```

Comment:
I'm not sure about this one. It is a server side rendered block, why we should disable the client side navigation?